2020/8xxx/CVE-2020-8563.json

```diff
@@ -1,18 +1,101 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
     "CVE_data_meta": {
+        "ASSIGNER": "security@kubernetes.io",
+        "DATE_PUBLIC": "2020-10-15T04:00:00.000Z",
         "ID": "CVE-2020-8563",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
+        "STATE": "PUBLIC",
+        "TITLE": "Secret leaks in logs for vSphere Provider kube-controller-manager"
     },
+    "affects": {
+        "vendor": {
+            "vendor_data": [
+                {
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "Kubernetes",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_affected": "<",
+                                            "version_value": "1.19.3"
```

Comment:
Are versions prior to v1.19.0 affected?

Reply:
not for this CVE, but it is for the others and I think is covered?